app/src/main/java/ch/epfl/skysync/navigation/HomeGraph.kt

```diff
@@ -99,6 +99,15 @@ fun NavGraphBuilder.homeGraph(
           confirmationScreen(navController, flightId, flightsViewModel)
         }
 
+    /*composable(
```

Comment:
I think you can already connect

Reply:
I think we need to create a userDetailsViewModel before that will give us a list of all the finished flights and fetch a user given a userId